src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
+const tableElemet = document.querySelector('table');
+
+['thead', 'tbody', 'tfoot'].forEach((section) => {
+  const rows = tableElemet.querySelector(section).rows;
```

Comment:
Ensure that the `querySelector` for each section (`thead`, `tbody`, `tfoot`) returns a valid element. If any of these sections do not exist in the table, this line could throw an error.